src/functions.php

```diff
@@ -1,23 +1,22 @@
 <?php
 
-function displayDreamDetails(array $dreams): string{
+function displayDreamDetails(array $dreams): string {
     $dreamOutput ='';
-    foreach(array_reverse($dreams) as $dream) {
+    $dreamsChronologically = array_reverse($dreams);
```

Comment:
so this is good but technically incorrect, chronological order is their original order from the db, whereas this is reverse chronological 😅 chronological means oldest to newest

Reply:
done now...